### PR TITLE
Free Tier release updates

### PR DESCRIPTION
**Pull request by natapg**

### CODE REVIEW

**Review by Lisenish — COMMENTED** [review]
Looks good, but we shouldn't merge it until the free-tier release

File: static/admin/config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 backend:
   name: git-gateway
-  branch: master
+  branch: release/free-tier
```
Comment (Lisenish) [inline static/admin/config.yml:3]: Don't forget to change back (I think Savannah finished with all required updates?

Comment (natapg) [reply static/admin/config.yml:3]: I'll change it before merging to developer, just in case we find something else to change

Comment (natapg) [reply static/admin/config.yml]: changed back

**Review by Lisenish — APPROVED** [review]

**Pull request merged**